component/espejote-templates/httproute-certificate-manager.jsonnet

```diff
@@ -0,0 +1,169 @@
+local esp = import 'espejote.libsonnet';
+
+local config = import 'httproute-certificate-manager/config.json';
+
+local createCertificateAnnotation = config.createCertificateAnnotation;
+local clusterIssuerAnnotation = config.clusterIssuerAnnotation;
+local issuerAnnotation = config.issuerAnnotation;
+local tlsSecretNameAnnotation = config.tlsSecretNameAnnotation;
+local gatewayDefaultClusterIssuerAnnotation = config.gatewayDefaultClusterIssuerAnnotation;
+
+local gateways = esp.context().gateways;
+
+local result = {
```

Comment:
My idea of error passing. Might be a bit overengineered, but i'm still experimenting what feels right 😅

Reply:
feels very Rust :)